opentech/apply/funds/templates/funds/includes/review_table.html

```diff
@@ -0,0 +1,22 @@
+{% load review_tags %}
+
+<table>
+    <tr>
+        <th>OTF</th>
+        <th>{{ object.reviews.staff_reccomendation|traffic_light }}</th>
+        <th>{{ object.reviews.staff_score|default_if_none:'-' }}</th>
+    </tr>
+    {% include 'funds/includes/review_table_row.html' with reviews=staff_reviews %}
+    {% include 'funds/includes/review_table_row.html' with reviews=object.missing_staff_reviews missing=True %}
+</table>
+
+<table>
+    <tr>
+        <th>Reviewers</th>
+        <th>{{ object.reviews.reviewers_reccomendation|traffic_light }}</th>
+        <th>{{ object.reviews.reviewers_score|default_if_none:'-' }}</th>
```

Comment:
If the user is added as reviewer, but is also staff, the block will display both...